libcontainer/SPEC.md

```diff
@@ -278,10 +301,26 @@ maximum memory bandwidth of 20% on socket 0 and 70% on socket 1.
 
 "linux": {
     "intelRdt": {
-        "closID": "guaranteed_group",
         "l3CacheSchema": "L3:0=7f0;1=1f",
         "memBwSchema": "MB:0=20;1=70"
-    }
+	}
+}
+```
+Another example:
+```
+We only want to monitor memory bandwidth and llc occupancy.
+"linux": {
+    "intelRdt": {
+	}
+}
+
+or
+
+"linux": {
+    "intelRdt": {
+        "l3CacheSchema": "",
+        "memBwSchema": ""
+	}
```

Comment:
In my opinion, this config interface may make users confused. The users don't know what it really implies: "monitoring only" or "empty allocation config"?

What do you think if we could add a flag into the config linux.intelRdt to indicate either CTRL_MON group or MON group will be created for the container? Something like: "monitoringOnly": true/false?

Reply:
I left a related comment.